packages/@tinacms/core/src/media.ts

```diff
@@ -144,6 +150,25 @@ export interface MediaList {
   nextOffset?: MediaListOffset
 }
 
+/**
+ * Custom action props for a media tab
+ */
+interface CustomActionProps {
+  cms: any
```

Comment:
What's this cms for? If it is what I think, that's more than what we need.

Reply:
Agreed. Overkill. Removed.